js/ui/drop_down_button.js

```diff
@@ -78,18 +78,18 @@ let DropDownButton = Widget.inherit({
             noDataText: formatMessage("dxCollectionWidget-noDataText"),
 
             /**
-             * @name dxDropDownButtonOptions.updateButtonOnSelection
+             * @name dxDropDownButtonOptions.useSelectMode
              * @type boolean
              * @default true
              */
-            updateButtonOnSelection: true,
+            useSelectMode: true,
```

Comment:
Alexander Bykov suggested these options to be set to false by default. Is it going to be done in a later PR?